packages/puppeteer-crawler/src/internals/puppeteer-launcher.ts

```diff
@@ -2,11 +2,11 @@ import ow from 'ow';
 import { Browser } from 'puppeteer';
 import { PuppeteerPlugin } from '@crawlee/browser-pool';
 import { BrowserLaunchContext, BrowserLauncher } from '@crawlee/browser';
+import log from '@apify/log';
 import { applyStealthToBrowser, StealthOptions } from './stealth';
 
 /**
  * The default user agent used by `Actor.launchPuppeteer`.
- * Last updated on 2020-05-22.
```

Comment:
why was this removed? seems to be still present in the js repo

https://github.com/apify/apify-js/blob/master/src/constants.js#L5